From: Jeremy Salwen <jeremysalwen@gmail.com>
Date: Sat, 28 May 2011 03:20:35 -0400
Subject: [PATCH] fix includes path and linkage.

---
 pitch.pro |    9 ++++-----
 1 files changed, 4 insertions(+), 5 deletions(-)

diff --git a/pitch.pro b/pitch.pro
index 4d644a5..a2dc46a 100644
--- a/pitch.pro
+++ b/pitch.pro
@@ -12,8 +12,8 @@ unix{
     MY_LIB_PATH += -L/Users/student/usr/local/lib
     MY_INCLUDE_PATH += /Users/student/usr/local/include
   }else{ #Linux
-    MY_LIB_PATH += -L/home/inferno/research/pitch/lib
-    MY_INCLUDE_PATH += /home/inferno/research/pitch/include/Qt /home/inferno/research/pitch/include/qwt /home/inferno/research/pitch/include
+    MY_LIB_PATH += -L/usr/lib
+    MY_INCLUDE_PATH += /usr/include/qt4 /usr/include/qwt-qt4 /usr/include
   }
 }
 win32{ #Windows
@@ -325,10 +325,9 @@ unix{
 
     DEFINES += LINUX
     INCLUDEPATH += $$MY_INCLUDE_PATH
-    LIBS += $$MY_LIB_PATH -lfftw3f -lqwt -lasound
+    LIBS += $$MY_LIB_PATH -lfftw3f -lqwt-qt4 -lasound
     CONFIG += warn_off
-    QMAKE_CXXFLAGS += -Wall -Wno-non-virtual-dtor
-    QMAKE_CXXFLAGS -= -g
+    QMAKE_CXXFLAGS += -Wall -Wno-non-virtual-dtor -pipe
     profile {
       QMAKE_CXXFLAGS += -pg
       QMAKE_LFLAGS += -pg
-- 
